avm/res/service-bus/namespace/main.bicep

```diff
@@ -187,22 +187,22 @@ module serviceBusNamespace_authorizationRules 'authorization-rule/main.bicep' =
   }
 }]
 
-module serviceBusNamespace_disasterRecoveryConfig 'disaster-recovery-config/main.bicep' = if (!empty(disasterRecoveryConfigs)) {
+module serviceBusNamespace_disasterRecoveryConfig 'disaster-recovery-config/main.bicep' = if (!empty(disasterRecoveryConfig)) {
   name: '${uniqueString(deployment().name, location)}-DisasterRecoveryConfig'
   params: {
     namespaceName: serviceBusNamespace.name
-    name: disasterRecoveryConfigs.?name ?? 'default'
-    alternateName: disasterRecoveryConfigs.?alternateName ?? ''
-    partnerNamespaceResourceID: disasterRecoveryConfigs.?partnerNamespace ?? ''
+    name: disasterRecoveryConfig.?name ?? 'default'
+    alternateName: disasterRecoveryConfig.?alternateName ?? ''
+    partnerNamespaceResourceID: disasterRecoveryConfig.?partnerNamespace ?? ''
   }
 }
 
-module serviceBusNamespace_migrationConfigurations 'migration-configuration/main.bicep' = if (!empty(migrationConfigurations)) {
+module serviceBusNamespace_migrationConfigurations 'migration-configuration/main.bicep' = if (!empty(migrationConfiguration)) {
   name: '${uniqueString(deployment().name, location)}-MigrationConfigurations'
   params: {
     namespaceName: serviceBusNamespace.name
-    postMigrationName: migrationConfigurations.?postMigrationName
-    targetNamespaceResourceId: migrationConfigurations.?targetNamespace
+    postMigrationName: migrationConfiguration.?postMigrationName ?? ''
+    targetNamespaceResourceId: migrationConfiguration.?targetNamespace ?? ''
```

Comment:
Is null as the default not working?

Reply:
In the UDT these are set to required strings as per the documentation:
![image](https://github.com/Azure/bicep-registry-modules/assets/48600046/8e8fcf90-1142-4a00-ba23-aab67248dc72)

Reply:
ok but in turn - if they're required shouldn't we make them required? Will the service now use actually `''` for the postMigrationName? Or does that trigger another default? Just curious

Reply:
If we do make them required, we get a BCP warning that they are currently null but expecting a string. The resource block only runs if the `migrationConfigurations` is not empty so we cannot complete them as a defualt parameter

Reply:
![image](https://github.com/Azure/bicep-registry-modules/assets/48600046/519ef679-7fb3-4557-a170-901c15478931)

Reply:
I shall reach out

Reply:
Pushed and updated - please reveiw @AlexanderSehr